js/offer.js

```diff
@@ -0,0 +1,80 @@
+import {getHouseRentalCount} from './random-object.js';
+const cardTemplate = document.querySelector('#card').content.querySelector('.popup');
+const templateFRAGMENT = document.createDocumentFragment();
+const announcementMap = document.querySelector('.map__canvas');
```

Comment:
it is not a map, dont lean on selectors when define node, use understandable names